status-im/status-react
# feat: add blur effect on scroll for buttons inside gallery view

fixes #16127 

### Summary

This PR adds blur effect on gallery view buttons and dropdown, and some UI adjustments to comply with Figma designs

#### Results

##### iOS

###### Light
https://github.com/status-im/status-mobile/assets/18485527/69995208-aaa6-4f20-b378-e42bb1f9f82d

###### Dark
https://github.com/status-im/status-mobile/assets/18485527/bc2d0185-0bea-4919-9c16-d9caa98b20db

##### Android

###### Light
https://github.com/status-im/status-mobile/assets/18485527/03f38569-48de-4a2b-8ffd-16f69fd80cbf

###### Dark
https://github.com/status-im/status-mobile/assets/18485527/9e7e6304-42f8-4936-8fff-726a5aa4507d

#### Platforms

- Android
- iOS

### Steps to test

- Open Status
- Open a chat
- Tap on send image button on composer
- Scroll through images and verify the blur inside buttons / dropdown

status: ready
